src/settings.ts

```diff
@@ -479,6 +502,7 @@ export const ALL_BURN_TEMPLATES: Array<VenueTemplate> = [
   VenueTemplate.audience,
   VenueTemplate.performancevenue,
   VenueTemplate.themecamp,
+  VenueTemplate.embeddable,
```

Comment:
[NB] While I don't believe `ALL_BURN_TEMPLATES` is used anywhere anymore anyway, looking at the other templates included in the list, not sure the `VenueTemplate.embeddable` needs to be part of them.

Reply:
removed in https://github.com/sparkletown/sparkle/commit/c95c932f2c2b10cfa98f09884f67ddfd551ced42